views/js/qtiCreator/widgets/helpers/formElement.js

```diff
@@ -282,7 +282,10 @@ define([
                     updateResponseDeclaration(element, value, options.updateCardinality);
                 }
 
-                if (!value && (element.is('orderInteraction') || element.is('graphicOrderInteraction'))) {
+                if ((this.name === 'upperBound') && this.disabled) {
+                    // if upperBound field triggered this callback and it is disabled, upperbound attribute should be removed.
+                    element[options.attrMethodNames.remove](name);
+                } else if (!value && (element.is('orderInteraction') || element.is('graphicOrderInteraction'))) {
```

Comment:
Maybe instead of add new condition we can update existing one?
You can ask UX why this is only for `orderInteraction` and `graphicOrderInteraction`
I think it should be removed if value is removed
```suggestion
                if (!value) {
```

Reply:
https://oat-sa.atlassian.net/wiki/spaces/IMS/pages/1204585768/Max+min+values+in+QTI+interactions

We should modify `upperBound` in the same manner that max is modified

Reply:
> Maybe instead of add new condition we can update existing one? 
We can't because this function is called to handle other callback changes. For example, `maxChoice`